<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qAgate</class>
 <widget class="QMainWindow" name="qAgate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>qAgate</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qagate.qrc">
    <normaloff>:/qagate.svg</normaloff>:/qagate.svg</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>7</number>
          </property>
          <property name="tabsClosable">
           <bool>false</bool>
          </property>
          <widget class="Home" name="home">
           <attribute name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/home/icons/home.svg</normaloff>:/home/icons/home.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>&amp;Home</string>
           </attribute>
          </widget>
          <widget class="Visuals" name="visuals">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/visuals/icons/eye.svg</normaloff>:/visuals/icons/eye.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>&amp;Visuals</string>
           </attribute>
          </widget>
          <widget class="MD" name="md">
           <attribute name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/MD/icons/md.svg</normaloff>:/MD/icons/md.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>&amp;MD</string>
           </attribute>
          </widget>
          <widget class="LocalMode" name="local">
           <attribute name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/local/icons/local.svg</normaloff>:/local/icons/local.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Local</string>
           </attribute>
          </widget>
          <widget class="DensityMode" name="density">
           <attribute name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/density/icons/density.svg</normaloff>:/density/icons/density.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Density</string>
           </attribute>
          </widget>
          <widget class="PhononsMode" name="phonons">
           <attribute name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/phonons/icons/phonons.svg</normaloff>:/phonons/icons/phonons.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>Phonons</string>
           </attribute>
          </widget>
          <widget class="Tools" name="tools">
           <attribute name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/tools/icons/tools.svg</normaloff>:/tools/icons/tools.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>&amp;Tools</string>
           </attribute>
          </widget>
          <widget class="Settings" name="settings">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="icon">
            <iconset resource="qagate.qrc">
             <normaloff>:/settings/icons/settings.svg</normaloff>:/settings/icons/settings.svg</iconset>
           </attribute>
           <attribute name="title">
            <string>&amp;Settings</string>
           </attribute>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="View" name="view">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TimeLine" name="timeLine" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MediaPlayer" name="mediaPlayer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="bottom" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="Logger" name="logger" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSecret">
   <property name="text">
    <string>Secret</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+J</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TimeLine</class>
   <extends>QWidget</extends>
   <header>gui/timeline.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MediaPlayer</class>
   <extends>QWidget</extends>
   <header>gui/mediaplayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>View</class>
   <extends>QOpenGLWidget</extends>
   <header>gui/view.h</header>
  </customwidget>
  <customwidget>
   <class>Logger</class>
   <extends>QWidget</extends>
   <header>gui/logger.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Settings</class>
   <extends>QWidget</extends>
   <header>tabs/settings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Visuals</class>
   <extends>QWidget</extends>
   <header>tabs/visuals.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Home</class>
   <extends>QWidget</extends>
   <header>tabs/home.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MD</class>
   <extends>QWidget</extends>
   <header>tabs/md.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tools</class>
   <extends>QWidget</extends>
   <header>tabs/tools.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LocalMode</class>
   <extends>QWidget</extends>
   <header>tabs/localmode.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DensityMode</class>
   <extends>QWidget</extends>
   <header>tabs/densitymode.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PhononsMode</class>
   <extends>QWidget</extends>
   <header>tabs/phononsmode.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qagate.qrc"/>
 </resources>
 <connections/>
</ui>
